--- origsrc/xcb-util-0.3.0/atom/Makefile.am	2008-09-15 14:00:09.000000000 -0500
+++ src/xcb-util-0.3.0/atom/Makefile.am	2008-09-22 02:55:01.020497800 -0500
@@ -10,7 +10,7 @@
 libxcb_atom_la_SOURCES = atoms.c atomlist.m4 atoms.gperf.m4 xcb_atom.h.m4
 libxcb_atom_la_CPPFLAGS = $(XCB_CFLAGS)
 libxcb_atom_la_LIBADD = $(XCB_LIBS)
-libxcb_atom_la_LDFLAGS = -version-info 1:0:0
+libxcb_atom_la_LDFLAGS = -version-info 1:0:0 -no-undefined
 
 pkgconfig_DATA = xcb-atom.pc
 
--- origsrc/xcb-util-0.3.0/aux/Makefile.am	2008-06-17 15:19:44.000000000 -0500
+++ src/xcb-util-0.3.0/aux/Makefile.am	2008-09-22 02:55:20.364247800 -0500
@@ -10,6 +10,7 @@
 libxcb_aux_la_SOURCES = xcb_aux.c
 libxcb_aux_la_CPPFLAGS = $(XCB_CFLAGS)
 libxcb_aux_la_LIBADD = $(XCB_LIBS)
+libxcb_aux_la_LDFLAGS = -no-undefined
 
 pkgconfig_DATA = xcb-aux.pc
 
--- origsrc/xcb-util-0.3.0/event/Makefile.am	2008-09-16 12:31:43.000000000 -0500
+++ src/xcb-util-0.3.0/event/Makefile.am	2008-09-22 02:55:25.770497800 -0500
@@ -10,7 +10,7 @@
 libxcb_event_la_SOURCES = event.c
 libxcb_event_la_CPPFLAGS = $(XCB_CFLAGS)
 libxcb_event_la_LIBADD = $(XCB_LIBS)
-libxcb_event_la_LDFLAGS = -version-info 1:0:0
+libxcb_event_la_LDFLAGS = -version-info 1:0:0 -no-undefined
 
 pkgconfig_DATA = xcb-event.pc
 
--- origsrc/xcb-util-0.3.0/icccm/Makefile.am	2008-09-15 13:40:53.000000000 -0500
+++ src/xcb-util-0.3.0/icccm/Makefile.am	2008-09-22 02:55:30.520497800 -0500
@@ -11,7 +11,7 @@
 libxcb_icccm_la_CPPFLAGS = $(XCB_CFLAGS) $(XCB_ATOM_CFLAGS) $(XCB_EVENT_CFLAGS) \
 	$(XCB_PROPERTY_CFLAGS)
 libxcb_icccm_la_LIBADD = $(XCB_LIBS) $(XCB_ATOM_LIBS) $(XCB_PROPERTY_LIBS)
-libxcb_icccm_la_LDFLAGS = -version-info 1:0:0
+libxcb_icccm_la_LDFLAGS = -version-info 1:0:0 -no-undefined
 
 pkgconfig_DATA = xcb-icccm.pc
 
--- origsrc/xcb-util-0.3.0/image/Makefile.am	2008-09-15 13:40:53.000000000 -0500
+++ src/xcb-util-0.3.0/image/Makefile.am	2008-09-22 02:55:42.254872800 -0500
@@ -11,6 +11,7 @@
 libxcb_image_la_SOURCES = xcb_image.c
 libxcb_image_la_CPPFLAGS = $(XCB_CFLAGS) $(XCB_SHM_CFLAGS) $(XCB_AUX_CFLAGS)
 libxcb_image_la_LIBADD = $(XCB_LIBS) $(XCB_SHM_LIBS) $(XCB_AUX_LIBS)
+libxcb_image_la_LDFLAGS = -no-undefined
 
 pkgconfig_DATA = xcb-image.pc
 
--- origsrc/xcb-util-0.3.0/keysyms/Makefile.am	2008-06-17 15:19:44.000000000 -0500
+++ src/xcb-util-0.3.0/keysyms/Makefile.am	2008-09-22 02:55:56.879872800 -0500
@@ -10,6 +10,7 @@
 libxcb_keysyms_la_SOURCES = keysyms.c
 libxcb_keysyms_la_CPPFLAGS = $(XCB_CFLAGS)
 libxcb_keysyms_la_LIBADD = $(XCB_LIBS)
+libxcb_keysyms_la_LDFLAGS = -no-undefined
 
 pkgconfig_DATA = xcb-keysyms.pc
 
--- origsrc/xcb-util-0.3.0/property/Makefile.am	2008-09-16 12:31:43.000000000 -0500
+++ src/xcb-util-0.3.0/property/Makefile.am	2008-09-22 02:56:01.551747800 -0500
@@ -10,7 +10,7 @@
 libxcb_property_la_SOURCES = property.c
 libxcb_property_la_CPPFLAGS = $(XCB_CFLAGS) $(XCB_EVENT_CFLAGS)
 libxcb_property_la_LIBADD = $(XCB_LIBS) $(XCB_EVENT_LIBS)
-libxcb_property_la_LDFLAGS = -version-info 1:0:0
+libxcb_property_la_LDFLAGS = -version-info 1:0:0 -no-undefined
 
 pkgconfig_DATA = xcb-property.pc
 
--- origsrc/xcb-util-0.3.0/renderutil/Makefile.am	2008-06-17 15:19:44.000000000 -0500
+++ src/xcb-util-0.3.0/renderutil/Makefile.am	2008-09-22 02:56:16.426747800 -0500
@@ -8,6 +8,7 @@
 libxcb_render_util_la_SOURCES = cache.c util.c glyph.c
 libxcb_render_util_la_CPPFLAGS = $(XCB_CFLAGS) $(XCB_RENDER_CFLAGS)
 libxcb_render_util_la_LIBADD = $(XCB_LIBS) $(XCB_RENDER_LIBS)
+libxcb_render_util_la_LDFLAGS = -no-undefined
 
 pkgconfig_DATA = xcb-renderutil.pc
 
--- origsrc/xcb-util-0.3.0/reply/Makefile.am	2008-09-16 12:31:43.000000000 -0500
+++ src/xcb-util-0.3.0/reply/Makefile.am	2008-09-22 02:56:23.457997800 -0500
@@ -12,7 +12,7 @@
 libxcb_reply_la_SOURCES = reply.c
 libxcb_reply_la_CPPFLAGS = $(XCB_CFLAGS)
 libxcb_reply_la_LIBADD = $(XCB_LIBS) -lpthread
-libxcb_reply_la_LDFLAGS = -version-info 1:0:0
+libxcb_reply_la_LDFLAGS = -version-info 1:0:0 -no-undefined
 
 pkgconfig_DATA = xcb-reply.pc
 
