--- origsrc/xcb-util-0.2/atom/Makefile.am	2007-03-02 16:18:34.000000000 -0600
+++ src/xcb-util-0.2/atom/Makefile.am	2007-08-08 06:53:19.000000000 -0500
@@ -9,6 +9,7 @@
 
 libxcb_atom_la_SOURCES = atoms.c atomlist.m4 atoms.gperf.m4 xcb_atom.h.m4
 libxcb_atom_la_CPPFLAGS = $(XCB_CFLAGS)
+libxcb_atom_la_LDFLAGS = -no-undefined
 libxcb_atom_la_LIBADD = $(XCB_LIBS)
 
 pkgconfig_DATA = xcb-atom.pc
--- origsrc/xcb-util-0.2/convenient/Makefile.am	2007-03-02 16:18:34.000000000 -0600
+++ src/xcb-util-0.2/convenient/Makefile.am	2007-08-08 06:54:02.843750000 -0500
@@ -9,6 +9,7 @@
 
 libxcb_aux_la_SOURCES = xcb_aux.c
 libxcb_aux_la_CPPFLAGS = $(XCB_CFLAGS)
+libxcb_aux_la_LDFLAGS = -no-undefined
 libxcb_aux_la_LIBADD = $(XCB_LIBS)
 
 pkgconfig_DATA = xcb-aux.pc
--- origsrc/xcb-util-0.2/event/Makefile.am	2007-03-02 16:18:34.000000000 -0600
+++ src/xcb-util-0.2/event/Makefile.am	2007-08-08 06:53:33.890625000 -0500
@@ -9,6 +9,7 @@
 
 libxcb_event_la_SOURCES = events.c
 libxcb_event_la_CPPFLAGS = $(XCB_CFLAGS)
+libxcb_event_la_LDFLAGS = -no-undefined
 libxcb_event_la_LIBADD = $(XCB_LIBS)
 
 pkgconfig_DATA = xcb-event.pc
--- origsrc/xcb-util-0.2/icccm/Makefile.am	2007-03-02 16:18:34.000000000 -0600
+++ src/xcb-util-0.2/icccm/Makefile.am	2007-08-08 06:54:14.515625000 -0500
@@ -10,6 +10,7 @@
 libxcb_icccm_la_SOURCES = icccm.c
 libxcb_icccm_la_CPPFLAGS = $(XCB_CFLAGS) $(XCB_ATOM_CFLAGS) $(XCB_EVENT_CFLAGS) \
 	$(XCB_PROPERTY_CFLAGS)
+libxcb_icccm_la_LDFLAGS = -no-undefined
 libxcb_icccm_la_LIBADD = $(XCB_LIBS) $(XCB_ATOM_LIBS) $(XCB_PROPERTY_LIBS)
 
 pkgconfig_DATA = xcb-icccm.pc
--- origsrc/xcb-util-0.2/image/Makefile.am	2007-03-02 16:18:34.000000000 -0600
+++ src/xcb-util-0.2/image/Makefile.am	2007-08-08 06:54:33.000000000 -0500
@@ -11,6 +11,7 @@
 
 libxcb_image_la_SOURCES = xcb_image.c
 libxcb_image_la_CPPFLAGS = $(XCB_CFLAGS) $(XCB_SHM_CFLAGS) $(XCB_AUX_CFLAGS)
+libxcb_image_la_LDFLAGS = -no-undefined
 libxcb_image_la_LIBADD = $(XCB_LIBS) $(XCB_SHM_LIBS) $(XCB_AUX_LIBS)
 
 pkgconfig_DATA = xcb-image.pc
--- origsrc/xcb-util-0.2/keysyms/Makefile.am	2007-03-02 16:18:34.000000000 -0600
+++ src/xcb-util-0.2/keysyms/Makefile.am	2007-08-08 06:54:41.531250000 -0500
@@ -9,6 +9,7 @@
 
 libxcb_keysyms_la_SOURCES = keysyms.c
 libxcb_keysyms_la_CPPFLAGS = $(XCB_CFLAGS)
+libxcb_keysyms_la_LDFLAGS = -no-undefined
 libxcb_keysyms_la_LIBADD = $(XCB_LIBS)
 
 pkgconfig_DATA = xcb-keysyms.pc
--- origsrc/xcb-util-0.2/property/Makefile.am	2007-03-02 16:18:34.000000000 -0600
+++ src/xcb-util-0.2/property/Makefile.am	2007-08-08 06:54:54.781250000 -0500
@@ -9,6 +9,7 @@
 
 libxcb_property_la_SOURCES = prop.c
 libxcb_property_la_CPPFLAGS = $(XCB_CFLAGS) $(XCB_EVENT_CFLAGS)
+libxcb_property_la_LDFLAGS = -no-undefined
 libxcb_property_la_LIBADD = $(XCB_LIBS) $(XCB_EVENT_LIBS)
 
 pkgconfig_DATA = xcb-property.pc
--- origsrc/xcb-util-0.2/renderutil/Makefile.am	2007-03-02 16:18:34.000000000 -0600
+++ src/xcb-util-0.2/renderutil/Makefile.am	2007-08-08 06:55:03.531250000 -0500
@@ -7,6 +7,7 @@
 
 libxcb_render_util_la_SOURCES = cache.c util.c glyph.c
 libxcb_render_util_la_CPPFLAGS = $(XCB_CFLAGS) $(XCB_RENDER_CFLAGS)
+libxcb_render_util_la_LDFLAGS = -no-undefined
 libxcb_render_util_la_LIBADD = $(XCB_LIBS) $(XCB_RENDER_LIBS)
 
 pkgconfig_DATA = xcb-renderutil.pc
--- origsrc/xcb-util-0.2/reply/Makefile.am	2007-03-02 16:18:34.000000000 -0600
+++ src/xcb-util-0.2/reply/Makefile.am	2007-08-08 06:55:15.734375000 -0500
@@ -11,6 +11,7 @@
 
 libxcb_reply_la_SOURCES = reply.c
 libxcb_reply_la_CPPFLAGS = $(XCB_CFLAGS)
+libxcb_reply_la_LDFLAGS = -no-undefined
 libxcb_reply_la_LIBADD = $(XCB_LIBS) -lpthread
 
 pkgconfig_DATA = xcb-reply.pc
